logistics/scripts/modules/path-resolver.Tests.ps1

```diff
@@ -0,0 +1,65 @@
+#  TO DO
+# use new-item instead of mkdir
```

Comment:
Please apply this TO DO

Reply:
removed these lines as they were not used anymore